src/components/views/rooms/RoomList2.tsx

```diff
@@ -200,6 +200,7 @@ export default class RoomList2 extends React.Component<IProps, IState> {
                     addRoomLabel={aesthetics.addRoomLabel}
                     isInvite={aesthetics.isInvite}
                     layout={this.state.layouts.get(orderedTagId)}
+                    showMessagePreviews={orderedTagId === DefaultTagID.DM}
```

Comment:
Should this be the case?

![image](https://user-images.githubusercontent.com/2403652/84089874-0f27e100-a9e8-11ea-992f-48276ee7ba1d.png)

Figma shows it on rooms too, wasn't it togglable per sublist?

Reply:
The feature is missing a setting and a whole lot of other work, which is why its a TODO and not implemented. Just needed something there to test the styling.

Reply:
also, for the running theme on these PRs: rubber stamped with *deferred*.